week4tasks/src/week4/reverse3/ReverseString.java

```diff
@@ -0,0 +1,14 @@
+package week4.reverse3;
+
+import java.util.Scanner;
+
+public class ReverseString {
```

Comment:
this class is like the class int this path week4tasks/src/week4/reerser2/revsers.java
so , delete one of them